.travis.yml

```diff
@@ -1,24 +1,33 @@
 language: python
-matrix:
-  include:
-    - python: 2.7
-      env: SYMPY_VERSION="oldest"
-    - python: 2.7
-      env: SYMPY_VERSION="latest"
-    - python: 2.7
-      env: SYMPY_VERSION="master"
-virtualenv:
-  system_site_packages: true
```

Comment:
Doesn't this flag have to stay in the file to ensure that the system packages are used? Travis defaults to a virtualenv that doesn't have access to the system packages.